dist/Caddyfile

```diff
@@ -0,0 +1,42 @@
+# This is a Caddyfile, i.e. a configuration file for Caddy.
+# Caddy (https://caddyserver.com) is a widely used open source web server,
+# which can replace Nginx as a reverse proxy in front of Mastodon.
+# If you decide to use Caddy with this configuration file, remember to replace
+# "example.com" with the URL of your Mastodon instance.
+
+example.com {
+	root * /home/mastodon/live/public
+
+	@staticfiles file
+	handle @staticfiles {
+		file_server
+	}
+
+	handle /api/v1/streaming* {
```

Comment:
I have more than one streaming container. Could we document in comments how to deal with this?
To <https://github.com/mastodon/mastodon/blob/877090518682b6c77ba9bdfa0231afd56daec44d/dist/nginx.conf#L10>, the equivalent would be:

~~~
        handle /api/v1/streaming* {
                reverse_proxy localhost:4000 localhost:4001 {
                        lb_policy least_conn
                        health_uri /api/v1/streaming/health
                }
        }
~~~